src/modules/governance/ProposalListItem.tsx

```diff
@@ -21,7 +22,53 @@ export function ProposalListItem({
     formatProposal(proposal);
   const { palette } = useTheme();
 
+  const delayedBridgeExecutors = [
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_ARBITRUM,
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_OPTIMISM,
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_POLYGON,
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_METIS,
+  ];
+  const lowercaseExecutors = delayedBridgeExecutors.map((str) => str.toLowerCase());
+
+  let proposalCrosschainBridge = false;
+
+  if (proposal.targets && proposal.targets.length > 0) {
+    const hasDelayedExecutor = proposal.targets.filter((address) =>
+      lowercaseExecutors.includes(address.toLowerCase())
+    );
+    if (hasDelayedExecutor.length > 0) {
+      proposalCrosschainBridge = true;
+    }
+  }
+
+  // Currently all cross-executors share this delay
+  // TO-DO: invetigate if this can be changed, if so, query on-chain
+  const twoDayDelay = 172800;
+
+  const executedL2 =
+    proposal.executionTime > 0
+      ? proposal.executionTime + twoDayDelay > proposal.executionTime
```

Comment:
Won't this always be true? Should it check if execution time plus delay is greater than now?

Reply:
Yeah this statement is wrong, updated to check that currentTimestamp > executionTimestamp: ` Math.floor(Date.now() / 1000) > proposal.executionTime + twoDayDelay`